packages/cozy-client/src/store/queries.js

```diff
@@ -198,7 +198,7 @@ const query = (state = queryInitialState, action, documents) => {
         response.meta && response.meta.count
           ? response.meta.count
           : response.data.length
-      if (action.backgroundFetching) {
+      if (action.backgroundFetching || action.invalidate) {
```

Comment:
**The principles I wanted to apply :**
- bypass the fetchPolicy
- run the query like normal ones, fetchStatus set as `loading` before returning `loaded`
- overwrites previous data like `backgroundFetching` when receive result

We could consider a combination with `backgroundFetching` to avoid the loading state and therefore the spinners and create a smoother transition.

The data is replaced because the function is exited before the data merge step performed [here](https://github.com/cozy/cozy-client/blob/ea8b7909ac8241fcc443fc16536985b702966e40/packages/cozy-client/src/store/queries.js#L213).